app/views/workshops/show_md_file.erb

```diff
@@ -1,5 +1,14 @@
 <% provide(:title, @title) %>
 
+<div class="modal fade" id="project-submission-form" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-body">
+        <%= render "new_project_form" %>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
I'd move this over to a partial because the partial's name will give context for what this is.